roles/openshift_health_checker/openshift_checks/logging/elasticsearch.py

```diff
@@ -40,8 +33,10 @@ def run(self, tmp, task_vars):
         # TODO(lmeyer): run it all again for the ops cluster
         return {"failed": False, "changed": False, "msg": 'No problems found with Elasticsearch deployment.'}
 
-    def _not_running_elasticsearch_pods(self, es_pods):
+    def not_running_elasticsearch_pods(self, es_pods):
         """Returns: list of running pods, list of errors about non-running pods"""
+        # FIXME: the docstring above is clearly wrong.
+        # I don't understand this method enough to fix the doscstring.
```

Comment:
Looks like we only use the method [to determine which pods are running](https://github.com/openshift/openshift-ansible/pull/4485#diff-cf30713e491fc112ba1639d63f6cb276R55) and to return a list of pods we should warn about being in a "not running" state. I think it would be better to have a `running_es_pods` method, and derive the list of non-running pods based on that